### PR TITLE
fix: add Vaibhav Upreti

### PR DESCRIPTION
**Pull request by VaibhavUpreti**
My linkfree profile

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/776"></a>

### CODE REVIEW

File: public/data/VaibhavUpreti.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        "icon": "youtube"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/vaibhav-upreti-739b0421a/",
+        "icon": "linkedin"
+      }
+    ],
+    "milestones": [
+
```
Comment (schmelto) [inline public/data/VaibhavUpreti.json:29]:
```suggestion
```

File: public/data/VaibhavUpreti.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        "icon": "youtube",
+        "color": "red",
+        "description": "First real video",
+        "url": "https://www.youtube.com/channel/UCmlGx1Tz_HimE9iO5wrJVdA"
+      },
+  
+      {
+        "title": "Joined MAIT BTech AI&ML",
+        "date": "2021",
+        "icon": "github",
```
Comment (Pradumnasaraf) [inline public/data/VaibhavUpreti.json:43]: Instead of `github` icon, I think there needs to be something else.
Other reviews will suggest

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
There are some more indentation changes too

File: public/data/VaibhavUpreti.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      {
+        "title": "Joined MAIT BTech AI&ML",
+        "date": "2021",
+        "icon": "https://github.com/VaibhavUpreti.png",
+        "color": "green",
+        "description": "Joined MAIT BTech AI&ML",
+        "url": "https://github.com/VaibhavUpreti"
+      }
+    ]
+  }
```
Comment (Pradumnasaraf) [inline public/data/VaibhavUpreti.json:33]:
```suggestion
{
  "name": "Vaibhav Upreti",
  "type": "personal",
  "bio": "Programmer | Developer | Coder | Student | Learner",
  "avatar": "https://github.com/VaibhavUpreti.png",
  "links": [
    {
      "name": "Follow me on GitHub",
      "url": "https://github.com/VaibhavUpreti",
      "icon": "https://github.com/VaibhavUpreti.png"
    },
    {
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/vaibhav__upreti",
      "icon": "https://twitter.com/vaibhav__upreti/photo"
    },
    {
      "name": "Follow me on LinkedIn",
      "url": "https://www.linkedin.com/in/vaibhav-upreti-739b0421a/",
      "icon": "linkedin"
    }
  ],
  "milestones": [
    {
      "title": "Joined MAIT BTech AI&ML",
      "date": "2021",
      "icon": "https://github.com/VaibhavUpreti.png",
      "color": "green",
      "description": "Joined MAIT BTech AI&ML",
      "url": "https://github.com/VaibhavUpreti"
    }
  ]
}
```

Comment (Pradumnasaraf) [inline public/data/VaibhavUpreti.json:33]:
```suggestion
{
  "name": "Vaibhav Upreti",
  "type": "personal",
  "bio": "Programmer | Developer | Coder | Student | Learner",
  "avatar": "https://github.com/VaibhavUpreti.png",
  "links": [
    {
      "name": "Follow me on GitHub",
      "url": "https://github.com/VaibhavUpreti",
      "icon": "https://github.com/VaibhavUpreti.png"
    },
    {
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/vaibhav__upreti",
      "icon": "https://twitter.com/vaibhav__upreti/photo"
    },
    {
      "name": "Follow me on LinkedIn",
      "url": "https://www.linkedin.com/in/vaibhav-upreti-739b0421a/",
      "icon": "linkedin"
    }
  ],
  "milestones": [
    {
      "title": "Joined MAIT BTech AI&ML",
      "date": "2021",
      "icon": "https://github.com/VaibhavUpreti.png",
      "color": "green",
      "description": "Joined MAIT BTech AI&ML",
      "url": "https://github.com/VaibhavUpreti"
    }
  ]
}
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]

File: public/data/VaibhavUpreti.json (hunk 3 above)
Comment (VaibhavUpreti) [reply public/data/VaibhavUpreti.json]: Thanks again!!

File: public/data/VaibhavUpreti.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      "url": "https://github.com/VaibhavUpreti",
+      "icon": "https://github.com/VaibhavUpreti.png"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vaibhav__upreti",
+      "icon": "https://twitter.com/vaibhav__upreti/photo"
+    },
+    {
+      "name": "Follow me on LinkedIn",
```
Comment (krishguptadev) [inline public/data/VaibhavUpreti.json:18]: Not sure if we can follow on LinkedIn 🤔 

```suggestion
      "name": "Connect with me on LinkedIn",
```

File: public/data/VaibhavUpreti.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Vaibhav Upreti",
+  "type": "personal",
+  "bio": "Programmer | Developer | Coder | Student | Learner",
+  "avatar": "https://github.com/VaibhavUpreti.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/VaibhavUpreti",
+      "icon": "https://github.com/VaibhavUpreti.png"
```
Comment (krishguptadev) [inline public/data/VaibhavUpreti.json:10]:
```suggestion
      "icon": "github"
```

File: public/data/VaibhavUpreti.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vaibhav__upreti",
+      "icon": "https://twitter.com/vaibhav__upreti/photo"
```
Comment (krishguptadev) [inline public/data/VaibhavUpreti.json:15]:
```suggestion
      "icon": "twitter"
```

File: public/data/VaibhavUpreti.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Joined MAIT BTech AI&ML",
+      "date": "2021",
+      "icon": "https://github.com/VaibhavUpreti.png",
+      "color": "green",
+      "description": "Joined MAIT BTech AI&ML",
+      "url": "https://github.com/VaibhavUpreti"
```
[the remark this answers is not in the document]
Comment (VaibhavUpreti) [reply public/data/VaibhavUpreti.json]: Thank you Krish. That was really helpful.

File: public/data/VaibhavUpreti.json [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/vaibhav-upreti-739b0421a/",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Joined MAIT BTech AI&ML",
+      "date": "2021",
+      "icon": "https://github.com/VaibhavUpreti.png",
```
Comment (krishguptadev) [inline public/data/VaibhavUpreti.json:27]:
```suggestion
      "icon": "book",
```

File: public/data/VaibhavUpreti.json [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Joined MAIT BTech AI&ML",
+      "date": "2021",
+      "icon": "book",
+      "color": "green",
+      "description": "Joined MAIT BTech AI&ML",
```
Comment (krishguptadev) [inline public/data/VaibhavUpreti.json:29]:
```suggestion
      "description": "Joined MAIT BTech AI&ML"
```

**Comment by VaibhavUpreti** [comment]
I guess this file is now ready to be merged.

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**